theano/gof/opt.py

```diff
@@ -1305,7 +1301,7 @@ def apply_mult_opts(node, fgraph, multiple_opts=False, prev_repl=None):
                 opt_start = time.time()
                 repl = opt.transform(node)
                 opt_finish = time.time()
-                self.time_opts[opt] += opt_start - opt_finish
+                self.time_opts[opt] += opt_finish - opt_start
```

Comment:
By mistake, i did the subtraction the otherway. Till today, I had thought it was because of the strings that i moved around while printing